src/metabase/api/util.clj

```diff
@@ -53,4 +54,13 @@
         headers   {"Content-Disposition" "attachment; filename=\"connection_pool_info.json\""}]
     (assoc (response/response {:connection-pools pool-info}) :headers headers, :status 200)))
 
+
+(api/defendpoint POST "/product-feedback"
+  "Send feedback about metabase"
+  [:as {{:keys [comments source email]} :body}]
+  {comments ms/NonBlankString
+   source ms/NonBlankString
+   email [:maybe ms/NonBlankString]}
+  (send-product-feedback comments source email))
```

Comment:
A lot of times what we'll do is something like

```clojure
(future (send-product-feedback comments source email))
api/generic-204-no-content
```

This makes it clear that the handler won't block, just does it's little side effect forwarding, and immediately returns. I don't think there's any info we need to give to someone here except "thanks".